packages/react/runtime/compat/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+export * from '@lynx-js/react';
+
+/**
+ * Starts a transition update, marking state updates in the callback as non-urgent
+ *
+ * Note: This is Preact's startTransition implementation, which differs fundamentally from React's:
```

Comment:
I'm not sure if we should have "Preact" in our document or not. /cc @hzy @Huxpro wdyt
```suggestion
 * Note: This implementation differs fundamentally from React's:
```